private/rejectvisual_channel.m

```diff
@@ -33,7 +33,7 @@
   ft_progress('init', cfg.feedback, 'filtering data');
   for i=1:ntrl
     ft_progress(i/ntrl, 'filtering data in trial %d of %d\n', i, ntrl);
-    [data.trial{i}, label, time, cfg.preproc] = preproc(data.trial{i}, data.label, data.time{i}, cfg.preproc);
+    [data.trial{i}, ~, ~, cfg.preproc] = preproc(data.trial{i}, data.label, data.time{i}, cfg.preproc);
```

Comment:
please don't use tildes for unused output arguments. this does not go well with octave.